### PR TITLE
Fix the gap between email and text

### LINKED ISSUE #153: The gap between the email and the text is very poor in the contact text
- The gap between the email and the text is very poor in the contact text
- Add a padding or margin to increase the gap

image

### PR DESCRIPTION
**Pull request by Rishav1707**
## Purpose

The purpose of this Pull Request is to fix #153 <issue-number>

## Goals

- Added the padding property to the email icon in style.css file.

### Before
206857804-2b43ac31-91f3-4793-9d3d-b6271a4bdd7a
### After
Screenshot 2022-12-11 094803

### Preview Link

https://pr-155-NEW-YOSHITHA-WEBSITE.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by Rishav1707** [comment]
Please run the workflow... @YoshithaRathnayake

**Comment by YoshithaRathnayake** [comment]
Yeah Good Job @Rishav1707 pls add a comment like you are working on it on the issue

File: dark/assets/css/styles.css [hunk 1]
```diff
@@ -1051,6 +1051,10 @@ html {
   z-index: -1;
 }
 
+.fa{
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1054]: Can you use bootstrap class instead of custom css @Rishav1707

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css (hunk 1 above)
Comment (Rishav1707) [reply dark/assets/css/styles.css:1054]: I will try once then I'll report to you.

Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:1054]: https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE/blob/bbb62296bd27270313436aa40de85fb2fb471136/dark/index.html#L242

Add px-1 inside the class tag @Rishav1707

Comment (Rishav1707) [reply dark/assets/css/styles.css:1054]: Sorry I am unable to use bootstrap class instead of custom CSS. @YoshithaRathnayake

Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:1054]: > https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE/blob/bbb62296bd27270313436aa40de85fb2fb471136/dark/index.html#L242
> 
> Add px-1 inside the class tag @Rishav1707

Here I already tought you no? @Rishav1707

Comment (Rishav1707) [reply dark/assets/css/styles.css:1054]: One min

Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:1054]: Just remove the custom css and add this px-1 to the class tag @Rishav1707 

> https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE/blob/bbb62296bd27270313436aa40de85fb2fb471136/dark/index.html#L242
> 
> Add px-1 inside the class tag @Rishav1707

Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:1054]: > One min

Sure

Comment (Rishav1707) [reply dark/assets/css/styles.css:1054]: Done

Comment (Rishav1707) [reply dark/assets/css/styles.css:1054]: In #156

**Pull request closed without merging**